otio_cookelensmetadata/adapter.py

```diff
@@ -68,6 +68,11 @@ def category(self) -> Category:
         except ValueError:
             return self.Category.UNKNOWN
 
+    @property
+    def is_kinematic(self) -> bool:
+         kinematic_strings = ["magnetometer", "accelerometer", "gyro"]
+         return any(substring in str(self) for substring in kinematic_strings)
```

Comment:
nitpick: Since this code is called a lot in long streams, there may be a minor performance improvement by making it a `set` and a constant on the class.

Perhaps just below `identifier` have something like:
`KINEMATIC_RECORD_TYPES = {"magnetometer", "accelerometer", "gyro"}`

Then you can use the test: `substring in self.KINEMATIC_RECORD_TYPES`

This approach avoids a possible re-allocation of a list on every call (it's pre-cached as part of the class) and utilizes the more performant string membership testing the `set` implementation provides. Usually I wouldn't bother with this kind of thing, but in the size of these datasets it might actually help a little.

Reply:
One other small thought, it might make sense to make this constant "private" by calling it `_KINEMATIC_RECORD_TYPES` instead, my preference is very weak about that though.

Reply:
Yes, let me adjust that a bit. All good ideas.

Reply:
I made the adjustment, and that should help performance as you say. My approach here is a bit of a brute-force string search, and maybe there's a more elegant way to do this, but it does work.

Reply:
Yeah, I'm sure there might be some more "optimal" way but I can't think of it immediately and like the readability and maintainability of this approach. Unless it's a really significant enhancement, I'd rather the machines spent the time on my slow understandable code than humans spend time trying to understand "fast" and hard to read code :)